### ISSUE TITLE
Mean IoU of 0.008, even when training to overfit.

### ISSUE THREAD

**Issue by ghost** [DrSleep/tensorflow-deeplab-resnet#194]
I use my own dataset, but I train using just one image. The loss goes down to as low as 0.004. But when I evaluate using the same image I use for training, I get a mean IoU of 0.008. Why could this be the case? I made sure my image and label was being read correctly.

**Comment by Simpatech-app** [comment]
I have a same issue the loss is going to a small value,but the mean IOU is very bad.
